src/splat/segtypes/common/c.py

```diff
@@ -380,13 +381,16 @@ def get_c_line_include_macro(
             # IDO uses the asm processor to embeed assembly, and it doesn't require a special directive to include symbols
             asm_outpath = asm_out_dir / self.name / f"{sym.filename}.s"
             rel_asm_outpath = os.path.relpath(asm_outpath, options.opts.base_path)
-            return f'#pragma GLOBAL_ASM("{rel_asm_outpath}")'
+            final_path = PureWindowsPath(f"{rel_asm_outpath}").as_posix()
```

Comment:
Why use `PureWindowsPath` here? Shouldn't the `.as_posix()` be enough?
```suggestion
            final_path = Path(f"{rel_asm_outpath}").as_posix()
```

Reply:
I was gonna say, could all `os.path` things be changed to use Path consistently?

Reply:
`Path` doesn't have all the functionalities that `os.path` have. iirc `relpath` is one of those, sadly.

Also, even if it would be nice to have, this kind of cleanup is a bit out of scope for this PR, we can handle that afterwards.

Reply:
I won't lie--my python skills are quite rusty. It's fixed now and also formatted properly.